### PR TITLE
Rm anndata

### PR DESCRIPTION
**Pull request by JuanluOnieva-biomage**
This PR is related to the jira ticket (https://biomage.atlassian.net/browse/BIOMAGE-459?atlOrigin=eyJpIjoiZDNlZDkyMmI5ZDA0NDdjMmI3MjZlZTRmNTJkMjlkY2YiLCJwIjoiaiJ9), which consists on cleaning up the data-ingest script from the Anndata generation step. 

packages removed from python side: {scanpy, anndata, louvain, leidenalg}

### CODE REVIEW

File: src/4-create-adata.py [hunk 1]
```diff
@@ -202,14 +93,14 @@ def main():
         ]
     )
 
+    experiment_id="58bec69943ed27c9ea1fffb46a4f2468"
```
Comment (ivababukova) [inline src/4-create-adata.py:96]: why is there a hardcoded experiment id?

File: src/4-create-adata.py [hunk 2]
```diff
@@ -15,96 +13,15 @@
...
-    )
-    gene_annotations = pandas.read_csv(
-        "/output/r-out-annotations.csv",
-        sep="\t",
-        # usecols=[3, 4, 5],
-        usecols=[3, 4],
-        index_col=0,
-        # names=["gene_ids", "gene_names", "descriptions"],
-        names=["gene_ids", "gene_names"],
-        na_values=["None"],
```
Comment (ivababukova) [inline src/4-create-adata.py:41]: do we have an equivalent of this written with seurat?

File: src/4-create-adata.py (hunk 1 above)
Comment (JuanluOnieva-biomage) [reply src/4-create-adata.py]: Sorry! During a test I forgot to rm.

File: src/4-create-adata.py (hunk 2 above)
Comment (JuanluOnieva-biomage) [reply src/4-create-adata.py:41]: Yes! We save in the slot "Misc" of Seurat a data frame with the genes dispersion info

File: requirements.txt [hunk 3]
```diff
@@ -1,10 +1,6 @@
 boto3
-scanpy
-anndata
 scvelo
 python-igraph
-louvain
 flake8
 cryptography
-leidenalg
```
Comment (ogibson-biomage) [inline requirements.txt:9]: I think r uses leidenalg from python to compute the leiden clusters. I know we don't currently do Leiden in the ingest though.

File: src/4-create-adata.py [hunk 4]
```diff
@@ -264,12 +153,8 @@ def main():
     )
     bucket, key = FILE_NAME.split("/", 1)
     
-    with open("/output/experiment.h5ad", "rb") as f:
-        s3.put_object(Body=f, Bucket=bucket, Key=key)
-    
-    print("uploading R object to s3...")
     with open("/output/experiment.rds", "rb") as f:
-        s3.put_object(Body=f, Bucket=bucket, Key=key.replace("python.h5ad", "r.rds"))
```
Comment (ogibson-biomage) [inline src/4-create-adata.py:272]: I don't understadn why the replace is no longer necessary?

File: src/4-create-adata.py [hunk 5]
```diff
@@ -264,12 +153,8 @@ def main():
...
+        s3.put_object(Body=f, Bucket=bucket, Key=key)
     
     print("successful. experiment is now accessible at:")
     print(f"https://scp.biomage.net/experiments/{experiment_id}/data-exploration")
```
Comment (ogibson-biomage) [inline src/4-create-adata.py:160]: I think the name of this file "create_adata" doesn't make sense any longer, and we should consider changing it.

**Review by ogibson-biomage — COMMENTED** [review]
This one seems pretty straightforward, if you could clarify the few comments with stuff I didn't understand I'll approve tomorrow!

**Review by seb-mueller — APPROVED** [review]
Just went through it with Juan and it looks fine

**Pull request merged**